test/libsolidity/syntaxTests/immutable/try_catch_initialized.sol

```diff
@@ -0,0 +1,25 @@
+contract A
+{
+    uint256 public immutable variable;
+
+    constructor()
+    {
+        B b;
+        try b.foo()
+        {
+            variable = 2;
+        }
+        catch {}
```

Comment:
Can you also add another test where it's inside the catch branch? Maybe even a specialized catch?

Reply:
Gonna cover all of them (in this test case though).